lte/gateway/c/oai/test/sgw_task/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+add_compile_options(-std=c++14)
+
+set(MME_APP_LIB_DIR $ENV{C_BUILD}/oai/tasks/mme_app)
+include_directories(${MME_APP_LIB_DIR})
+
+include_directories("${PROJECT_SOURCE_DIR}")
+
+add_library(SGW_TASK_TEST_LIB
```

Comment:
Please use the `SPGW_TASK_TEST_LIB` to avoid confusion with the new SGW_S8 task. Also, rename the directory to `test/spgw_task`. Eventually, we should also rename the `tasks/sgw` directory to `tasks/spgw`.

Reply:
resolved
will file a task for renaming the directory!